src/add.js

```diff
@@ -0,0 +1,18 @@
+import { isValidNumbers } from "./validate.js";
+
+const add = {
+    sumNumbers ({ arrayNumbers }) {
+        console.log(arrayNumbers)
+        arrayNumbers = arrayNumbers.map(num => Number(num));
```

Comment:
일관성과 추후 변경사항 최소화를 고려할 때 인자를 괄호로 감싸는 게 좋을 것 같습니다:) [8.4](https://github.com/ParkSB/javascript-style-guide#arrows--one-arg-parens)
```suggestion
        arrayNumbers = arrayNumbers.map((num) => Number(num));
```

혹은 아래처럼 간결하게 표현할 수도 있습니다. Number 함수에 num이 바로 인자로 들어갑니당.
```suggestion
        arrayNumbers = arrayNumbers.map(Number);
```